app/views/admin/configs/show.html.erb

```diff
@@ -1045,7 +1045,7 @@
           </div>
         <% end %>
 
-        <%= form_for(SiteConfig.new, url: admin_config_path) do |f| %>
+        <%= form_for(Settings::RateLimit.new, url: admin_settings_rate_limits_path) do |f| %>
```

Comment:
**(question):** Would it make sense to pull these forms out into partials whilst we're splitting up the settings?

Reply:
Yeah, so I thought about this repeatedly while working on this PR but didn't do it so far because I wasn't sure if we want to pull out each section or only the ones that got refactored? I also wasn't sure if it will conflict with your admin restructure work and the last thing I wanted was to create even more merge conflicts than I already had to deal with 😄 But I'm definitely open to revisit this during the end when I do the final cleanup.